sky/backends/backend_utils.py

```diff
@@ -1407,10 +1407,10 @@ def _get_tpu_vm_pod_ips(ray_config: Dict[str, Any],
         tpuvm_json = json.loads(stdout)
         if tpuvm_json['state'] != 'READY':
             # May be a leaked preempted resource, or terminated by user in the
-            # console.
+            # console, or still in the process of being created.
             ux_utils.console_newline()
             logger.warning(f'TPU VM {tpu_id} is not in READY state. '
-                           'Skipping...')
+                           'Skipping IP query...')
```

Comment:
I feel "skipped querying its IP" being printed during down still raises the question that whether it's actually being down'd. When do we need this warning to be printed? Is it safe ti change it to debug?

Reply:
Good point. actually @Michaelvll asked the same question before. The reason why we decided to make it a warning message is because normally this is not supposed to happen.
When it happens, it indicates there's likely a resource leak so we think it worth printing the message to users. I clarify the msg a bit. if still not helpful we can change it to debug.